test/ports/rtubufferedport.test.js

```diff
@@ -84,6 +84,230 @@ describe('Modbus RTU buffered port', function() {
                 });
             });
         });
+        
+                it('Special data package, should return a valid Modbus RTU message', function(done) {
+            port.once('data', function(data) {
+                expect(data.toString('hex')).to.equal('010380018301830183018301830183018301830183018301830183018301830183018301830183018301830183018301830183018301830183018301830183018301830183018301830183018301830183018301830183018301830183018301830183018301830183018301830183018301830183018301830183018301830183018346e0');
+                done();
+            });
+            port.open(function() {
+                port.write(new Buffer('010300000040443A', 'hex'));
+                setTimeout(function() {
+                    port._client.receive(new Buffer('01', 'hex'));
```

Comment:
it will be nicer if this values where in a constant, and you will use a loop to send them ...
you can even have a helper function for that if you want ...

```
LONG_MODBUS = '0103 ....'
...
expect(data.toString('hex')).to.equal(LONG_MODBUS)
...
for (i = 0; i < LONG_MODBUS.length; i += 2) {
    ...
    port._client.receive(new Buffer(LONG_MODBUS.slice(i, i + 2), 'hex'));
```